### DISCUSSION TITLE
How to pack to .GOB afterwards?

### DISCUSSION THREAD

**Discussion by indiasker** [smlu/ProjectMarduk#12] [Q&A]
Hello! I stumbled upon your tool via the thread in the gog-discussion. Since I am a severa arachnophobe too, it's exactly what I am looking for: Removing spiders from the game.
So I downloaded your tool, extracted the .GOB-files and editet the files actor_spider and actor_stickyspider as you mentioned.
But what to do afterwards? I guess I have to pack it back to a .GOB-file? But how would I do that?

**Comment by smlu** [comment]
Hey, no you don't need to pack the COG script files back to GOB file. Just put them in the `<IM_Installl>\Resource\cog` folder and the game will pick them. Also note that since the scripts were modified you'll have to start the game from the start.
